lib_nbgl/src/nbgl_fonts.c

```diff
@@ -772,10 +772,12 @@ uint16_t nbgl_getTextNbLinesInWidth(nbgl_font_id_e fontId,
                 if (last_nb_lines < nbLines) {
                     last_nb_lines = nbLines;
                 }
+#ifndef SCREEN_SIZE_WALLET
```

Comment:
Better use #ifdef SCREEN_SIZE_NANO than #ifndef SCREEN_SIZE_WALLET

Unfortunately it's not enough, the "if (with != 0)" must also be surrounded with #ifdef SCREEN_SIZE_NANO... but why is the interrest of this condition ?

Reply:
The change related to **_if (nbLines == 4)_** is only intended to nanos devices when generating screenshots: thanks for the **#ifdef SCREEN_SIZE_NANO**, I will use it.
The **_if (with != 0)_** is intended to fix an issue about how nb lines is computed: I can make it nano devices specific, if you want, but I think we need to change the way the number of lines is computed and this is the first step in that direction.

Reply:
let's discuss it later but yes, make it Nano Specific because it creates a bug in NOMINAL Stax/Flex UX

Reply:
Ok, I reverted the changes I made with **if (with != 0)** unless where it is necessary for LNX/LNS+ and we'll discuss about that later.